types/fastify-user-agent/package.json

```
@@ -0,0 +1,24 @@
+{
+    "private": true,
+    "name": "@types/fastify-user-agent",
+    "version": "1.0.9999",
+    "projects": [
+        "https://github.com/Eomm/fastify-user-agent"
+    ],
+    "dependencies": {
+        "@types/node": "*",
+        "@types/useragent": "^2.3.0",
+        "fastify": "^4.0.0",
+        "fastify-user-agent": "^1.0.0",
+        "useragent": "^2.3.0"
```

```suggestion
        "@types/node": "*",
        "@types/useragent": "^2.3.0",
        "fastify": "^4.0.0"
```

You shouldn't be adding a dep to `fastify-user-agent`; that's _this_ pacakge.

`useragent` doesn't have types, you already listed `@types/useragent` above.

Delete both of these and your PR will pass.